fxa-auth-db-server/test/backend/db_tests.js

```diff
@@ -1208,6 +1211,36 @@ module.exports = function(config, DB) {
                 t.equal(device.callbackPublicKey, '', 'callbackPublicKey updated')
                 t.equal(device.callbackAuthKey, '', 'callbackAuthKey updated')
 
+                // Make the device a zombie, by giving it a non-existent session token
```

Comment:
@philbooth While I am not entirely sure how the zombie session gets created, would it make sense to have some checks to `updateDevice` to ensure session is valid before updating?